main/settings_course_etl.py

```diff
@@ -105,3 +105,16 @@
 TIKA_TIMEOUT = get_int("TIKA_TIMEOUT", 60)
 TIKA_OCR_STRATEGY = get_string("TIKA_OCR_STRATEGY", "no_ocr")
 SKIP_TIKA = get_bool("SKIP_TIKA", default=False)
+
+
+# Base content URLs for different sources
+CONTENT_BASE_URL_MITXONLINE = get_string(
```

Comment:
will these values also work fine for rc? (do we need an associated PR in ol-infrastructure?)